pd-control.md

```diff
@@ -326,10 +326,20 @@ Usage:
 
 - `enable-debug-metrics` is used to enable the metrics for debugging. When you set it to `true`, PD enables some metrics such as `balance-tolerant-size`.
 
-- `enable-placement-rules` is used to enable placement rules.
+- `enable-placement-rules` is used to enable placement rules, which is enabled by default in v5.0 and later versions.
 
 - `store-limit-mode` is used to control the mode of limiting the store speed. The optional modes are `auto` and `manual`. In `auto` mode, the stores are automatically balanced according to the load (experimental).
 
+- PD will round the number of flow information to reduce the update of statistics caused by changes in Region flow information. This configuration item is used to specify the number of digits to round the Region flow information. For example, the flow `100512` will be rounded to `100500` because the default value is `3`. This configuration replaces `trace-region-flow`. 
```

Comment:
```suggestion
- PD will round the number of flow information to reduce the update of statistics caused by changes in Region flow information. This configuration item is used to specify the number of digits to round the Region flow information. For example, the flow `100512` will be rounded to `101000` because the default value is `3`. This configuration replaces `trace-region-flow`. 
```